docker/journald/Dockerfile

```diff
@@ -4,27 +4,27 @@ FROM ${BUILD_IMAGE}
 
 ARG UID
 
-STOPSIGNAL SIGRTMIN+3
+RUN <<-EOF
```

Comment:
First of all, applause of using the `<<-` marker, then we don't see leading spaces!